modules/core/shared/src/main/scala/net/Protocol.scala

```diff
@@ -209,7 +209,7 @@ object Protocol {
     readTimeout:  Duration
   ): Resource[F, Protocol[F]] =
     for {
-      bms <- BufferedMessageSocket[F](host, port, 256, debug, sg, socketOptions, sslOptions, readTimeout) // TODO: should we expose the queue size?
```

Comment:
This change didn't appear to make a difference so not sure we should touch it at this point.